### ISSUE TITLE
Import Lua config doesn't work at first try.

### ISSUE THREAD

**Issue by ddisanti** [mpcjanssen/simpletask-android#682]
Playing with @smichel17's debug build, I noted that ```Import Lua config``` doesn't work when applied. The edit box stays empty. Exiting Simpletask and open it again doesn't work either. You have to kill Simpletask to get Lua config loaded.
The same applies to Cloudless 8.3.0.

**Comment by smichel17** [comment]
Just to clarify, which of these are the steps to reproduce?

1. 
  - Import lua config
  - force quit simpletask
  - Open lua config; find it imported
2. 
  - Import lua config
  - force quit simpletask
  - Open lua config
  - Import lua config; works

**Comment by smichel17** [comment]
On my galaxy S4, current (buggy) behavior is:

- Open lua config and import
  - Import not shown on screen
- close lua config via top left X
- reopen lua config. Import is successful

Or

- Open lua config and import
  - Import not shown on screen
- close lua config via save button
- reopen lua config. Import is not successful

---

Behavior should be:

- Import lua config -> Screen immediately shows changes
- close via X -> import is discarded
- close via save -> Import is saved

**Comment by smichel17** [comment]
I believe what's happening is that the import writes directly to the app settings, but doesn't update the text box, so when you hit the save button, it overwrites your newly imported lua config with whatever is in the text box (your old config).

**Comment by ddisanti** [comment]
@smichel17 Lua config behaves as you described in your second comment and expected behaviour should be as you said.
Just for fun, I use Lua config to apply a theme according the time of the day and also to adjust the font size of the list. This is what I use:
 
    function theme()
      hora = tonumber(os.date("%H"))
      if (hora > 6 and hora < 19) then
        return "dark"
      else
        return "black"
      end
    end

    function tasklistTextSize() return 13.5 end

    -- Variables globales
    dia = 86400

The thing is that the theme and font  settings are applied only after killing Simpletask from the Recents Android screen.

**Comment by smichel17** [comment]
The part where the theme does not update unless you close the app from recent is a pretty involved fix and deserves its own issue.

**Comment by mpcjanssen** [comment]
Theme updates are only applied when an activity is recreated. It's hard to force an activity to recreate based on Lua code.

**Comment by smichel17** [comment]
The original issue from this thread is solved.
